setup.py

```diff
@@ -51,10 +51,11 @@ def main():
           download_url='https://github.com/apel/ssm/releases',
           license='Apache License, Version 2.0',
           install_requires=[
-              'stomp.py<5.0.0', 'python-ldap', 'argo-ams-library',
+              'stomp.py<5.0.0', 'python-ldap',
           ],
           extras_require={
-              'python-daemon': ['python-daemon'],
+              'AMS': ['argo-ams-library'],
+              'daemon': ['python-daemon'],
```

Comment:
Changing things?! I guess as this is milestone'd for 3.0.0 we can go crazy :p!

Reply:
The dictionary key is meant to be the name of the optional "feature" of your software rather than just the name of the package.